php/class-delivery.php

```diff
@@ -153,6 +153,49 @@ function ( $post ) {
 		}
 	}
 
+	/**
+	 * Maybe filter out Cloudinary URLs in post meta.
+	 *
+	 * @param null|bool $check      Whether to allow adding metadata for the given type.
+	 * @param int       $object_id  The ID of the object metadata is for.
+	 * @param string    $meta_key   The Metadata key.
+	 * @param mixed     $meta_value Metadata value.
+	 *
+	 * @return null|bool
+	 */
+	public function maybe_filter_out_metadata( $check, $object_id, $meta_key, $meta_value ) {
+
+		$current       = current_filter();
+		$is_serialized = false;
+		$is_object     = is_object( $meta_value );
+
+		if ( ! is_string( $meta_value ) ) {
+			$is_serialized = true;
+			$meta_value    = wp_json_encode( $meta_value );
```

Comment:
@pereirinha  - I think we should have a check here to be sure that it was encoded.

Reply:
This is a great point. Thanks. I've updated the code.